metricbeat/module/php_fpm/php_fpm.go

```diff
@@ -0,0 +1,103 @@
+package php_fpm
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+const (
+	defaultScheme = "http"
+	defaultPath   = "/status"
+)
+
+// HostParser is used for parsing the configured php-fpm hosts.
+var HostParser = parse.URLHostParserBuilder{
+	DefaultScheme: defaultScheme,
+	DefaultPath:   defaultPath,
```

Comment:
To define a config option to change the `defaultPath` you can use `PathConfigKey`. See https://github.com/elastic/beats/blob/master/metricbeat/module/prometheus/collector/collector.go#L25 as an example. I would suggest to use `path` or `status_path` as the config option. All other parts are done automatically by the HostParser.